### PR TITLE
Iac devops changes

### PR DESCRIPTION
**Pull request by cloudmlops**
Iac devops changes

### CODE REVIEW

File: 01c-s3.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  Type: String
+
+Resources:
+  S3Bucket:
+    Type: 'AWS::S3::Bucket'
+    Properties:
+      BucketName: !Join [ "-", [ !Ref EnvironmentName, !Ref BucketNameParam ] ]
+      AccessControl: PublicRead
+      WebsiteConfiguration:
+        IndexDocument: index12345.html
```
Comment (SHUBHAM-KALEKAR) [inline 01c-s3.yml:22]: this property shoulde be index.html

**Review by SHUBHAM-KALEKAR — CHANGES_REQUESTED** [review]
Modifiaction should be made in yml file as per comments.

File: 01c-s3.yml (hunk 1 above)
Comment (cloudmlops) [reply 01c-s3.yml]: changes are made as per review comments

**Review by SHUBHAM-KALEKAR — APPROVED** [review]
changed looks fine.

**Pull request merged**